index.html

```diff
@@ -14,87 +14,15 @@
         <div class="container__title">
             <h1>TO DO PLAN</h1>
         </div>
+        <div class="container__filter">
+            <button class="button-all button-pressed">All</button>
```

Comment:
let's remove the event listener for all of these buttons, just create a new function and call it from here.
Example:
```suggestion
            <button onclick="filterTodos('all')" class="button-all button-pressed">All</button>
```